ts/Grid/Core/Table/Actions/RowsVirtualizer.ts

```diff
@@ -469,21 +471,32 @@ class RowsVirtualizer {
      * once on initialization.
      */
     private getDefaultRowHeight(): number {
-        const mockRow = makeHTMLElement('tr', {
+        const vp = this.viewport;
+        const columns = vp.columns || {};
+        const mockHTMLRow = makeHTMLElement('tr', {
             className: Globals.getClassName('rowElement'),
             style: {
                 position: 'absolute'
             }
-        }, this.viewport.tbodyElement);
+        }, vp.tbodyElement);
+        const mockRow = new TableRow(vp, 0);
+        const mockCells = [];
+
+        for (let i = 0, iEnd = columns.length; i < iEnd; ++i) {
+            mockCells.push(new TableCell(mockRow, columns[i]));
+            mockCells[i].value = columns[i].data?.[0] || 'mock';
+            makeHTMLElement('td', {
+                innerHTML: mockCells[i].formatCell(),
+                className: Globals.getClassName('mockedCell')
+            }, mockHTMLRow);
+        }
 
-        const mockCell = makeHTMLElement('td', {
-            innerText: 'mock',
-            className: Globals.getClassName('mockedCell')
-        }, mockRow);
+        const defaultRowHeight = mockHTMLRow.offsetHeight;
 
-        const defaultRowHeight = mockRow.offsetHeight;
-        mockRow.remove();
-        mockCell.remove();
+        // Clear all mocked elements
+        mockCells.forEach(cell => cell.destroy());
```

Comment:
Destroying a row causes all cells that belong to it to be destroyed as well, so there is no point in doing it twice.